packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -565,6 +566,75 @@ describe("Integration tests", () => {
         );
       });
     });
+
+    describe("deleteTenantMailById", async () => {
+      const mailId = generateId();
+      const notDeletedMailId = generateId();
+
+      const tenant: Tenant = {
+        ...mockTenant,
```

Comment:
```suggestion
        ...getMockTenant(),
```

Reply:
I will replace the `mock` with their functions in the test split PR